conf/uvek.conf.part

```diff
@@ -1725,15 +1725,15 @@ source src_ch_bfe_waermepotential_gewaesser : def_searchable_features
          from bfe.waermepotential_gewaesser
  }
 
-source src_ch_bazl_gebietsbeschraenkungen_drohnen : def_searchable_features
+source src_ch_bazl_einschraenkungen_drohnen_zone_name : def_searchable_features
```

Comment:
to be consistent with the other indexes, please use the layer name as source.
`src_ch_bazl_einschraenkungen_drohnen`